spec/requests/organization_requests_spec.rb

```diff
@@ -58,6 +75,24 @@
                                             url: organization.url
                                           )
       end
+
+      context "when enable_packs flipper is on" do
+        it "should display custom units and units form" do
+          Flipper.enable(:enable_packs)
+          get edit_organization_path
+          expect(response.body).to include("Custom request units used (please use singular form -- e.g. pack, not packs)")
+          expect(response.body).to include unit.name
+        end
+      end
+
+      context "when enable_packs flipper is off" do
+        it "should not display custom units and units form" do
+          Flipper.disable(:enable_packs)
+          get edit_organization_path
+          expect(response.body).to_not include("Custom request units used (please use singular form -- e.g. pack, not packs)")
+          expect(response.body).to_not include unit.name
```

Comment:
Can we specify the unit name rather than relying on generated data?

Reply:
Done